dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/api/imports/HarvestedJsonParser.java

```diff
@@ -101,7 +102,7 @@ private DatasetVersion parseDatasetVersion(JsonObject obj, DatasetVersion dsv) t
             if (versionStateStr != null) {
                 dsv.setVersionState(DatasetVersion.VersionState.valueOf(versionStateStr));
             }
-            dsv.setReleaseTime(parseDate(obj.getString("releaseDate", null)));
+            dsv.setReleaseTime(parseDate(obj.getString("releaseTime", null)));
```

Comment:
There's a similar mismatch in `JsonParser`: https://github.com/CeON/dataverse/blob/develop/dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/util/json/JsonParser.java#L270

While I was able to verify here that we use `releaseTime` instead of `releaseDate` in the harvest export/import process, I'm not sure about the clients to the above one, which among other places, is exposed through the API.